src/processing/processors/processing_processor_mnn_posenet.cpp

```diff
@@ -107,71 +110,79 @@ namespace sgns::processing
         }
     }
 
-    std::unique_ptr<MNN::Tensor> MNN_PoseNet::MNNProcess(const std::vector<uint8_t>& imgdata, const int origwidth,
-        const int origheight, const std::string filename) {
-        std::vector<uint8_t> ret_vect(imgdata);
-
-        // Get Target Width
-        const int targetWidth = static_cast<int>((float)origwidth / (float)OUTPUT_STRIDE) * OUTPUT_STRIDE + 1;
-        const int targetHeight = static_cast<int>((float)origheight / (float)OUTPUT_STRIDE) * OUTPUT_STRIDE + 1;
-
-        // Scale
-        CV::Point scale;
-        scale.fX = (float)origwidth / (float)targetWidth;
-        scale.fY = (float)origheight / (float)targetHeight;
+    std::unique_ptr<MNN::Tensor> MNN_PoseNet::MNNProcess( const std::vector<uint8_t> &imgdata,
```

Comment:
Debatable. Because we are passing by value, adding `const` would not matter as much as if we were using references. Still, adding `const` would prevent mutability for those values, which is desired in this situation.